src/services/syncService.js

```
@@ -0,0 +1,147 @@
+    // Import game boards
+    if (userData.gameBoards && userData.gameBoards.length > 0) {
+      for (const board of userData.gameBoards) {
+        await upsertBoard({
+          title: board.title,
+          tiles: board.tiles,
+          tags: board.tags || [],
+          gameMode: board.gameMode || 'online',
+          isActive: board.isActive || 0,
+        });
+      }
+      console.log(userData.gameBoards?.length + ' game boards imported');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)